interview-questions.md

```diff
@@ -4,57 +4,57 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What does CRUD stand for?
 
-  Your answer:
+  Your answer:CRUD stands for: C-> Create R->Read U ->Update D ->Delete. These each are associated with web pages and they refelct what is used between the user and what the developer has created for that webpage. 
 
-  Researched answer:
+  Researched answer: Related to user interface and the users ability to store and retrieve data on a webpage. As a developer, we should be able to create products that at least meet this for the user. 
 
 
 
 2. What are the 5 HTTP verbs?
 
-  Your answer:
+  Your answer: Get retrieves info and displays it. Patch lets you update entries as a user. 
 
-  Researched answer:
+  Researched answer: The 5 HTTP verbs are Get, Head, Post, Put, and Delete. Get requests information. Head returns only the header information. A POST request is a statement by the client that a new “subordinate resource” should be created beneath the given URI. In practice, that means that POST requests should be used for things like adding a new post to a messageboard thread, adding a new post to a blog; or sending a message via a contact form. PUT lets the user update a section of the website. EX: updating a blog post would be done through a PUT request.
 
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
-  Your answer:
+  Your answer:You would need to extend the class from Component. Have a render with no argument and then in the return have a div tag with an <h1> tag containing the "Hello World!" within it. After closing the div brackets and the remaining class brackets. You would have to export the new class. Lastly, you would start a local server on your machine to render the <h1> message through the created React App.
 
-  Researched answer:
+  Researched answer: I think my answer goes over the necessary steps. 
```

Comment:
yep, this question was a little confusing and your answer is great. but just so you know, we were looking for: import, class name extends, render, return, div/React.Fragment would be a bonus, export included in this answer somehow. so good work.